fluentd/configs.d/openshift/output-es-config.conf

```diff
@@ -18,9 +18,9 @@
       type_name com.redhat.viaq.common
       retry_tag "retry_es"
 
-      # there is currently a bug in the es plugin + excon - cannot
-      # recreate/reload connections
-      reload_connections false
+      reload_connections "#{ENV['ES_RELOAD_CONNECTIONS'] || 'true'}"
+      reload_after "#{ENV['ES_RELOAD_AFTER'] || '100'}"
```

Comment:
We may want to consider including a 'unit' to explicitly identify what it is for clarity: '_OPERATIONS'

Reply:
I think we could get away with just adding a comment above this or expanding our docs or both

Reply:
I put in a link to the upstream docs - ok?

Reply:
wfm